keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -115,56 +112,68 @@ Proposed addition to core v1 API:
 type ServiceInternalTrafficPolicyType string
 
 const (
-	ServiceInternalTrafficPolicyTypeCluster     ServiceInternalTrafficPolicyType = "Cluster"
-	ServiceInternalTrafficPolicyTypePreferLocal ServiceInternalTrafficPolicyType = "PreferLocal"
-	ServiceInternalTrafficPolicyTypeLocal       ServiceInternalTrafficPolicyType = "Local"
+	ServiceTrafficPolicyTypeCluster     ServiceTrafficPolicyType = "Cluster"
+	ServiceTrafficPolicyTypeTopology    ServiceTrafficPolicyType = "Topology"
+	ServiceTrafficPolicyTypePreferLocal ServiceTrafficPolicyType = "PreferLocal"
+	ServiceTrafficPolicyTypeLocal       ServiceTrafficPolicyType = "Local"
 )
 
 // ServiceSpec describes the attributes that a user creates on a service.
 type ServiceSpec struct {
 	...
 	...
 
-	// internalTrafficPolicy denotes if the internal traffic for a Service should route
-	// to cluster-wide endpoints or node-local endpoints. "Cluster" routes internal traffic
-	// to a Service to all cluster-wide endpoints. "PreferLocal" will route internal traffic
-	// to node-local endpoints if one exists, otherwise it will fallback to the same behavior
-	// as "Cluster". "Local" routes traffic to node-local endpoints only, traffic is dropped
-	// if no node-local endpoints are ready.
-	InternalTrafficPolicy ServiceInternalTrafficPolicyType `json:"internalTrafficPolicy,omitempty"`
+	// trafficPolicy denotes if the traffic for a Service should route
+	// to cluster-wide endpoints or node-local endpoints. "Cluster" routes traffic
+	// to a Service to all cluster-wide endpoints. "Topology" routes traffic based on
```

Comment:
Nit: I'd personally prefer "PreferZone" over "Topology" here but no reason to block this PR.